src/Build/Resources/Strings.resx

```diff
@@ -158,6 +158,12 @@
   <data name="BuildFinishedFailure" xml:space="preserve">
     <value>Build FAILED.</value>
   </data>
+  <data name="BuildFinishedQuestionFailure" xml:space="preserve">
+    <value>Question build Failed.  The build exited early upon detection of an incremental issue.</value>
```

Comment:
```suggestion
    <value>Question build Failed. The build exited early upon detection of an issue with build incrementality.</value>
```

The original message felt like there is an issue that's incremental.
I'm not a native speaker though.

Reply:
Reworded to use "up-to-date" instead of incremental.